swift/example_code/iam/AttachRolePolicy/Tests/AttachRolePolicyTests/AttachRolePolicyTests.swift

```diff
@@ -124,16 +124,18 @@ final class AttachRolePolicyTests: XCTestCase {
 
     func testAttachRolePolicy() async throws {
         do {
+            print("Creating role")
             let testRole = try await createTestRole()
 
             // Add some policies to the role.
 
+            print("Creating policies")
             var testPolicyARNs: [String] = []
             for _ in 1...5 {
                 let policyArn = try await createTestPolicy()
                 testPolicyARNs.append(policyArn)
             }
-
+            print("Attachking policy")
```

Comment:
Attachking -> Attaching (as cool as an Attachking sounds)

Reply:
Ironically, the more important issue is that this function has three `print` statements that were only there for debugging purposes and need to be removed. Thanks for spotting this!